tutorials/build_system/prerequisites.md

```diff
@@ -2,122 +2,201 @@
+* There is no explicit `buildscript{}` in the file.
+  The IntelliJ IDEA Gradle plugin dynamically creates a `buildscript{}`.
+* Two plugins to Gradle are explicitly declared:
+  * The [Gradle Java](https://docs.gradle.org/current/userguide/java_plugin.html) plugin.
+  * The [IntelliJ IDEA Gradle plugin](https://github.com/JetBrains/gradle-intellij-plugin/).
+* The _Group ID_ from the Wizard [Project Naming Screen](#project-naming-screen) is the `(project.)group` value.
+* The _Version_ from the Wizard [Project Naming Screen](#project-naming-screen) is the `(project.)version` value.
+* The `sourceCompatibility` line is injected to enforce using JDK 1.8.
+* The only comment in the file is a link to the `README.md` for the IntelliJ IDEA Gradle plugin, which is a reference for the DSLs defined by the plugin.
+* The value of the Setup DSL attribute `intellij.version` specifies the version of the IntelliJ Platform to be used to build the plugin.
+  It defaults to the version of IntelliJ IDEA that was used to run the Wizard.
+* The value of the Patching DSL attribute `patchPluginXml.changeNotes` is set to place holder text.
+
+```text
+  plugins {
+      id 'java'
+      id 'org.jetbrains.intellij' version '0.4.8'
+  }
+  
+  group 'com.your.company'
+  version '1.0' 
+  sourceCompatibility = 1.8
+  
+  repositories {
+      mavenCentral()
+  } 
+  dependencies {
+      testCompile group: 'junit', name: 'junit', version: '4.12'
+  }
+  
+  // See https://github.com/JetBrains/gradle-intellij-plugin/
+  intellij {
+      version '2019.1'
+  }
+  patchPluginXml {
+      changeNotes """
+        Add change notes here.<br>
+        <em>most HTML tags may be used</em>"""
+  }
+```
 
-> **Note** Please make sure to always upgrade to latest version of `gradle-intellij-plugin` plugin, follow releases on [GitHub](https://github.com/JetBrains/gradle-intellij-plugin/releases). 
+### Opening a Wizard-Generated Gradle Project for the First Time
+The first time a project is opened an _Import Project from Gradle_ dialog appears.
+This dialog is essentially the same as the Wizard [Gradle Settings Screen](#gradle-settings-screen).
+Opening the project is an import because the Gradle plugin translates some of the project's Gradle files into IntelliJ IDEA project working files.
+For example, the import creates a `.idea` directory and a `<project_name>.iml` file.
+
+These IntelliJ IDEA project working files can be saved with the project to avoid the import step in the future.
+See the note at the end of the [Creating a new Gradle project](https://www.jetbrains.com/help/idea/gradle.html#project_create_gradle) about how newly created or imported projects are stored externally by default.
+
+## Adding Gradle Support to an Existing DevKit-Based IntelliJ Platform Plugin
+As of this writing, there isn't an IntelliJ IDEA Wizard to automatically convert a DevKit-based plugin project to a Gradle-based plugin project.
```

Comment:
and there probably never will be. please reword accordingly.

Reply:
Ok, I will set expectations accordingly.  It's a shame, because such a tool will be in demand by 3rd party developers as we move Gradle-based plugins further to the fore.

All that said, Is this procedure correct? Or is it better (safer) to just create a new Gradle project and then migrate source code?

Reply:
both ways should work